contrib/world_builder/CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ cmake_minimum_required(VERSION 3.13.4)
 
 project(WorldBuilder C CXX)
 
+message (STATUS "")
```

Comment:
It is not ideal if we need to modify the GWB CMakeLists.txt. Is it possible to put the status message in ASPECT's CMakeLists.txt?

Reply:
Sure, can do. That duplicates some information I want to provide as a section header, but that's fine.